assets/src/dashboard/app/views/shared/storyGridView.js

```diff
@@ -95,6 +96,12 @@ const StoryGridView = ({
     items: stories,
   });
 
+  useEffect(() => {
+    if (!activeGridItemId && returnStoryFocusId) {
+      itemRefs.current?.[returnStoryFocusId]?.children[0].focus();
```

Comment:
Would it ever not have a child at index 0? My guess is the story would be broken if not.

Reply:
nope! this will always always always have a child at index 0 if the children exist on the item key. we can check against it just incase.

Reply:
(ok i added a check, can never be too careful)